packages/patternfly-4/react-core/src/components/Tooltip/__snapshots__/Tooltip.test.tsx.snap

```diff
@@ -9,16 +9,12 @@ exports[`tooltip renders 1`] = `
       className="pf-c-tooltip"
       role="tooltip"
     >
-      <TooltipArrow
-        className={null}
-      />
-      <TooltipContent
-        className={null}
-      >
+      <Unknown />
+      <Unknown>
```

Comment:
Any idea why the snapshot updated to Unknown?

Reply:
yes, I saw that as well, I do not know, I thought it might have to do with the unit test using `const view = shallow(...`